.travis.yml

```diff
@@ -10,6 +10,5 @@ before_script:
   - if [[ -r /etc/hosts ]]; then cat /etc/hosts ; fi
   - cp config/database.yml.example config/database.yml
   - bundle exec rails generate argo:solr
```

Comment:
isn't this line also a no-op since `ci` calls `argo:install` which calls `rails g argo:solr`?